core/src/main/java/org/polypheny/db/algebra/AlgShuttle.java

```diff
@@ -76,7 +76,7 @@
  */
 public interface AlgShuttle {
 
-    AlgNode visit( Scan scan );
+    AlgNode visit( RelScan<?> scan );
```

Comment:
Ist there a reason why it is `LogicalDocumentScan` and `LogicalLpgScan` but in the relational case `RelScan`.

Reply:
Adjusted.